MastodonSDK/Sources/MastodonCore/Service/API/APIService+ListTimeline.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Combine
+import CoreData
+import CoreDataStack
+import MastodonSDK
+
+extension APIService {
+    public func listTimeline(
+        id: String,
+        query: Mastodon.API.Timeline.PublicTimelineQuery,
+        authenticationBox: MastodonAuthenticationBox
+    ) async throws -> Mastodon.Response.Content<[Mastodon.Entity.Status]> {
+        let domain = authenticationBox.domain
+        let authorization = authenticationBox.userAuthorization
+        
+        let response = try await Mastodon.API.Timeline.list(
+            session: session,
+            domain: domain,
+            query: query,
+            id: id,
+            authorization: authorization
+        ).singleOutput()
```

Comment:
Why do we need this to be a publisher (we don't use)? Is it to be consistent with everything else?

Reply:
Yes this was purely because it's done everywhere like this, but I agree that using streaming Publishers for one-off HTTPS¥ requests doesn't make much sense and it's a good opportunity for refactoring, so I've changed this.